playgound/cartpole.py

```diff
@@ -100,7 +101,16 @@ def hill_climbing(env):
 #
 # print(" ------------------------ ")
 
+print(" \n\n\n\n\n ")
+print("  -------------------------------- ")
+print(" |  WELCOME SHIDFAR's PLAYGROUND  |")
```

Comment:
Who is Shidfar adn what this playground is all about?

Reply:
its me and the comment was a test